midi2/src/utility.rs

```diff
@@ -260,6 +304,19 @@ mod tests {
         DeltaClockstamp::try_from(&[0x0040_0000][..]).unwrap();
     }
 
+    #[test]
+    fn delta_clock_stamp_u20() {
+        use crate::ux::u20;
+
+        let dc = DeltaClockstamp::try_from(&[0x004A_BCDE][..]).unwrap();
+        assert_eq!(dc.time_data(), u20::new(0xABCDE));
+
+        let buffer: [u32; 4] = [0, 0, 0, 0];
+        let mut dc = DeltaClockstamp::try_new_with_buffer(buffer).unwrap();
+        dc.set_time_data(u20::new(1_048_575));
+        assert_eq!(dc.time_data(), u20::new(0xF_FFFF));
```

Comment:
Perhaps let's use a less special value for this test case - i.e. not the value with all the bytes set high.

Reply:
Fixed in commit c9c7480